From f006ee36bec73cc129dab5a8d43cec77c10aef4e Mon Sep 17 00:00:00 2001
From: Wieland Hoffmann <themineo@gmail.com>
Date: Mon, 4 Jan 2016 14:43:44 +0100
Subject: [PATCH] Correctly add artist credits

---
 sir/wscompat/convert.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sir/wscompat/convert.py b/sir/wscompat/convert.py
index 56ba6cd..98eb37e 100644
--- a/sir/wscompat/convert.py
+++ b/sir/wscompat/convert.py
@@ -131,8 +131,8 @@ def convert_artist_credit(obj, include_aliases=True):
     :type obj: :class:`mbdata.models.ArtistCredit`
     """
     ac = models.artist_credit()
-    (ac.add_name_credit(convert_name_credit(nc, include_aliases)) for nc in
-     obj.artists)
+    [ac.add_name_credit(convert_name_credit(nc, include_aliases)) for nc in
+     obj.artists]
     return ac
 
 
